src/features/applications/update-applications.ts

```diff
@@ -21,7 +21,7 @@ export const updateApplicationInfo = (
   options?: ReadyActionExecutorOptions
 ) => readyActionExecutor(new UpdateApplicationInfoAction(client), options);
 
-class UpdateApplicationInfoAction extends InstructionsReadyAction {
+export class UpdateApplicationInfoAction extends InstructionsReadyAction {
```

Comment:
Would be nice to not have to export this. Might mean tests need to interact at a higher level. That might also alleviate the need to mock that higher level tho.